python/tvm/autotvm/tuner/xgboost_cost_model.py

```diff
@@ -144,6 +146,12 @@ def __init__(
         self._sample_size = 0
         self._reset_pool(self.space, self.target, self.task)
 
+    def _initializer(space, target, task):
```

Comment:
decorate it with `@staticmethod`

Reply:
I am moving this out of `XGBoostCostModel` class for a linting issue. Right now this function would be defined at line 351.